### PR TITLE
lint and tidy of desktop scss

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* linted and tidied up the _desktop.scss

## QA

1. go to every page in the /desktop section and see that it looks the same as live

### CODE REVIEW

File: static/css/section/_desktop.scss [hunk 1]
```diff
@@ -811,12 +593,12 @@ $item_margin: 40px;
     }
 
     @media only screen and (min-width: $breakpoint-large) {
```
Comment (nottrobin) [inline static/css/section/_desktop.scss]: Could you move these two `@media` statements so they're both inside `.four-col::after`?

File: static/css/section/_desktop.scss [hunk 2]
```diff
@@ -1,258 +1,153 @@
...
-      margin-top: 7%;
-    }
-  }
-
-  //Large screen
-  @media only screen and (min-width: $breakpoint-large) {
-    .row-hero {
+    @media only screen and (min-width: $breakpoint-large) {
       background-size: cover;
       margin-top: 0;
```
Comment (nottrobin) [inline static/css/section/_desktop.scss]: This doesn't need repeating, already declared in `min-width: $breakpoint-medium`.

File: static/css/section/_desktop.scss [hunk 3]
```diff
@@ -1,258 +1,153 @@
...
       min-height: 574px;
```
Comment (nottrobin) [inline static/css/section/_desktop.scss]: This is only 1 pixel different from what's declared in `min-width: $breakpoint-medium`. Is that necessary? If so, fair enough. But could we remove the duplicated `h1 { margin-top: 40px }` statement below this?

File: static/css/section/_desktop.scss [hunk 4]
```diff
@@ -1,258 +1,153 @@
...
-      font-size: 1em;
-      line-height: 1.5em;
+@charset 'UTF-8';
+
+// ---------------------------------------------------------------
+// ---------------------------------------------------------------
+//
+//  Project:		Ubuntu Front-End Cloud section
+//  Author:			Web Team at Canonical Ltd
+//  Last edited by: Peter Mahnke
```
Comment (nottrobin) [inline static/css/section/_desktop.scss]: As with https://github.com/ubuntudesign/www.ubuntu.com/pull/870#r79427677, would it be okay to remove "Last edited by" and "Last edited on"?

File: static/css/section/_desktop.scss [hunk 5]
```diff
@@ -1,258 +1,153 @@
...
-  overflow: hidden;
-  padding-bottom: 20px;
-
-  @media only screen and (min-width: $breakpoint-medium) {
-    padding-bottom: 0;
 
-    .pull-right {
-      max-width: 60%;
+    @media only screen and (min-width: $breakpoint-medium) {
+      > h2:first-child {
```
Comment (anthonydillon) [inline static/css/section/_desktop.scss]: This seems very specific. I wonder if the mark up could be improved or improved by adding a descriptive class?

File: static/css/section/_desktop.scss [hunk 6]
```diff
@@ -1,258 +1,153 @@
...
-    display: block;
-  }
+  .row-apps {
 
-  .box {
-    border: 0;
-  }
+    @media only screen and (max-width: $breakpoint-medium) {
+      overflow: hidden;
+      width: 100%;
```
Comment (anthonydillon) [inline static/css/section/_desktop.scss]: Don't rows always have 100% width?

File: static/css/section/_desktop.scss [hunk 7]
```diff
@@ -1,258 +1,153 @@
...
 
-  .where-to-find-us {
-    background: #fff;
-    margin: 0;
-    padding: 15px 20px;
+      .one-col {
```
Comment (anthonydillon) [inline static/css/section/_desktop.scss]: Really dislike styling grid classes. Is there something better we could do here?

File: static/css/section/_desktop.scss [hunk 8]
```diff
@@ -1,258 +1,153 @@
...
+        width: 64%;
+      }
     }
 
-    a { padding: 2px 0 2px 26px; }
-
-    .ask-ubuntu a {
-      background: url('#{$asset-server}7da3c811-icon-irc.png') 0 2px no-repeat;
+    // medium and up
+    @media only screen and (min-width: $breakpoint-medium) {
```
Comment (anthonydillon) [inline static/css/section/_desktop.scss]: Can you move this media query inside the element if it's a single selector.

File: static/css/section/_desktop.scss [hunk 9]
```diff
@@ -262,74 +157,88 @@
...
     @media only screen and (min-width: $breakpoint-medium) {
       background-image: url('#{$asset-server}d037640f-desktop.png?w=984');
     }
   }
 
   .row-gaming {
+
     @media only screen and (min-width: $breakpoint-medium) {
-      width: 100%;
       padding: 0;
```
Comment (anthonydillon) [inline static/css/section/_desktop.scss]: `padding: 0 0 55%` and remove the line below.

File: static/css/section/_desktop.scss [hunk 10]
```diff
@@ -607,200 +418,171 @@
...
+    margin-left: 0;
+    margin-right: 0;
 
-  .desktop-ubuntu-kylin,
-  .desktop-ubuntu-kylin-zh-CN {
-    .row-hero {
-      background: url('#{$asset-server}b0f9da06-desktop-kylin-hero.jpg?w=800') 498px 20px no-repeat;
-      padding-bottom: 55px;
+    @media only screen and (max-width: $breakpoint-medium) {
+      margin:0;
```
Comment (anthonydillon) [inline static/css/section/_desktop.scss]: Space after the `:`

File: static/css/section/_desktop.scss (hunk 1 above)
Comment (anthonydillon) [inline static/css/section/_desktop.scss]: Nice spot

File: static/css/section/_desktop.scss (hunk 2 above)
Comment (pmahnke) [inline static/css/section/_desktop.scss]: removed

File: static/css/section/_desktop.scss (hunk 3 above)
Comment (pmahnke) [inline static/css/section/_desktop.scss]: removing... agreed

File: static/css/section/_desktop.scss (hunk 1 above)
Comment (pmahnke) [inline static/css/section/_desktop.scss]: fixed... missed that.

File: static/css/section/_desktop.scss (hunk 4 above)
Comment (pmahnke) [inline static/css/section/_desktop.scss]: done... and changed 'cloud' to 'desktop'

File: static/css/section/_desktop.scss (hunk 5 above)
Comment (pmahnke) [inline static/css/section/_desktop.scss]: pretty sure it isn't even overriding anything... so removing

File: static/css/section/_desktop.scss (hunk 6 above)
Comment (pmahnke) [inline static/css/section/_desktop.scss]: removed

File: static/css/section/_desktop.scss (hunk 7 above)
Comment (pmahnke) [inline static/css/section/_desktop.scss]: I added more BEM classes and gave the icon a bit more space.

File: static/css/section/_desktop.scss (hunk 8 above)
Comment (pmahnke) [inline static/css/section/_desktop.scss]: done

File: static/css/section/_desktop.scss (hunk 10 above)
Comment (pmahnke) [inline static/css/section/_desktop.scss]: done.

File: static/css/section/_desktop.scss (hunk 9 above)
Comment (pmahnke) [inline static/css/section/_desktop.scss]: done.

**Comment by anthonydillon** [comment]
## QA
/desktop/education
- The "A global success in education" has collapsed

/desktop/government
- The "Used by governments across the globe" has collapsed

/desktop/partners
- The "Intel® Compute Stick with Ubuntu" section has lost its external icon on its link

**Comment by anthonydillon** [comment]
## QA 
### Small screen
/desktop
- "Looks great on the latest devices" section should have margin-bottom on the image
- The "Available on a huge range of hardware" section has to much padding at the top

/desktop/features
- "A whole world of apps" section looks different from live

/desktop/education
- "A global success in education" title has to much padding bottom
- "Countless open source tools and apps available" looks different from live

/desktop/government
- "Used by governments across the globe" title has to much padding bottom

**Comment by pmahnke** [comment]
/desktop - fixed
/desktop/features and education app grids - different and a bit better I think... I added some more classes to not overwrite the columns....
/desktop/education and government case studies - added media query for margin-bottom

**Comment by anthonydillon** [comment]
Looks good :+1: . Thank you for your continued contribution on this project.

**Pull request merged**